Dockerfile

```diff
@@ -64,7 +64,7 @@ RUN pip install --upgrade pip && \
     pip install -U sphinx-rtd-theme==0.1.9 recommonmark && \
     pip install pre-commit 'requests==2.9.2' 'ipython==5.3.0' && \
     pip install 'ipykernel==4.6.0' 'jupyter==1.0.0' && \
-    pip install rarfile
+    pip install rarfile 'scipy>=0.19.0' 'nltk>=3.2.2'
```

Comment:
Because we don't pip install `paddlepaddle` before testing, so it didn't track the dependencies.